etc/devices/fpga100t/Pin_Table.csv

```diff
@@ -0,0 +1,9 @@
+Group,Bump/Pin Name,Ball Name,Ball ID,Bump center_x,Bump center_y,Ball center_x,Ball center_y,X,X,X,X,Usable,Ref clock,Bank,ALT Function,Debug Mode,Scan Mode,Mbist Mode,Type,Direction,Voltage,Power Pad,Discription,Voltage2,Remark
```

Comment:
Is this csv temp till we have the final one?

Reply:
this is like template